WcaOnRails/app/models/competition.rb

```diff
@@ -515,8 +521,12 @@ def has_entry_fee?
     base_entry_fee.nonzero?
   end
 
+  def has_competitor_limit?
+    competitor_limit_flag
```

Comment:
Ahh, I did not think about that. That's a decent argument for allowing a null `competitor_limit`, sorry @pingskills.